packages/core/stories/multiline-input/multiline-input.stories.tsx

```diff
@@ -104,26 +106,32 @@ export const Disabled: ComponentStory<typeof MultilineInput> = (args) => {
 
 export const CharacterCount: ComponentStory<typeof MultilineInput> = (args) => {
   const [value, setValue] = useState<string>("Value");
+  const [isError, setIsError] = useState<boolean>(false);
   const MAX_CHARS = 10;
 
   const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
     const newVal = event.target.value;
+    setValue(newVal);
 
-    if (newVal.length <= MAX_CHARS) {
-      const value = newVal;
-      setValue(value);
+    if (newVal.length > MAX_CHARS) {
+      setIsError(true);
+    } else if (isError) {
+      setIsError(false);
```

Comment:
Wouldn't this just be:
```suggestion
      setIsError(newVal.length > MAX_CHARS);
```